crates/codegen/tests/integration_test.rs

```diff
@@ -0,0 +1,29 @@
+use std::path::PathBuf;
+
+use anyhow::Result;
+use javy_codegen::{Generator, LinkingKind, Plugin, JS};
+
+#[test]
+fn test_empty() -> Result<()> {
+    // Load valid JS from file.
+    let js = JS::from_file(
+        PathBuf::from(env!("CARGO_MANIFEST_DIR"))
+            .join("tests")
+            .join("sample-scripts")
+            .join("empty.js")
+            .as_path(),
+    )?;
+
+    // Load existing pre-initialized Javy plugin.
+    let plugin =
+        Plugin::new_from_path(PathBuf::from(env!("CARGO_MANIFEST_DIR")).join("test_plugin.wasm"))?;
```

Comment:
I was hoping you would use the default plugin as emitted by `javy emit-plugin` instead of the test plugin. Re-reading https://github.com/bytecodealliance/javy/pull/890/files#r1977733306 I'm realizing I was not explicit about using the default plugin (just `emit-plugin`). Specifically if we want to run the generated Wasm, it's easier with the default plugin because it doesn't have the extra non-WASIp1 import. It's not a big deal though.